test/integration/knative_test.go

```diff
@@ -35,11 +35,11 @@ const (
 
 // knativeMinKubernetesVersion indicates the minimum Kubernetes version
 // required in order to successfully run Knative tests.
-var knativeMinKubernetesVersion = semver.MustParse("1.21.0")
+var knativeMinKubernetesVersion = semver.MustParse("1.22.0")
 
 func TestKnativeIngress(t *testing.T) {
 	if clusterVersion.LT(knativeMinKubernetesVersion) {
-		t.Skip("knative tests can't be run on cluster versions prior to 1.21")
+		t.Skip("knative tests can't be run on cluster versions prior to 1.22")
```

Comment:
fixes unnecessary duplication of the magic number

```suggestion
		t.Skip("knative tests can't be run on cluster versions prior to", knativeMinKubernetesVersion)
```